src/lib/config/polygon/tokenlists.ts

```diff
@@ -3,9 +3,9 @@ import { TokenListURLMap } from '@/types/TokenList';
 const tokenlists: TokenListURLMap = {
   Balancer: {
     Default:
-      'https://raw.githubusercontent.com/balancer/tokenlists/main/generated/balancer.tokenlist.json',
+      'https://raw.githubusercontent.com/balancer/tokenlists/main/generated/listed-old.tokenlist.json ',
```

Comment:
Each of the URLs have this space at the end of the string, please remove.

Reply:
ah( fixed